.github/workflows/typechecks.yml

```diff
@@ -0,0 +1,28 @@
+name: Typecheck Tests
+
+on:
+  push:
+    paths:
+      - 'packages/UI/.storybook/**'
+      - 'packages/UI/src/**'
+
+defaults:
+  run:
+    working-directory: packages/UI
+
+jobs:
+  test:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    if: github.event.deployment_status.state == 'success'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+      - name: Install dependencies
+        run: yarn
+      - name: Run typecheck tests
+        run: yarn typecheck
+        env:
+          TARGET_URL: '${{ github.event.deployment_status.target_url }}'
```

Comment:
type checking doesn't require target url, since it runs without a deployed app. typescript does compilation typechecking and not run time one

Reply:
I was not sure about that, I understand now, thank you!